src/Microsoft.SqlTools.SqlCore/Microsoft.SqlTools.SqlCore.csproj

```diff
@@ -12,6 +12,11 @@
 		<PreserveCompilationContext>true</PreserveCompilationContext>
 		<AssemblyTitle>SqlTools SqlCore Library</AssemblyTitle>
 		<Description>Provides core sql functionality for SQL server editors like Object explorer, Query Execution and Scripting</Description>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>8.0</LangVersion>
```

Comment:
Could you move this to Directory.Build.Props?

Reply:
I think the rest of the projects are using lang version 11 (default in net core 7). The default for net standard 2.0 is 7.3 which does not support nullable types so I am forcing that here.